bin/ycsb

```diff
@@ -43,40 +46,46 @@ DATABASES = {
     "mongodb-async": "com.yahoo.ycsb.db.AsyncMongoDbClient",
     "nosqldb"      : "com.yahoo.ycsb.db.NoSqlDbClient",
     "orientdb"     : "com.yahoo.ycsb.db.OrientDBClient",
-    "redis"        : "com.yahoo.ycsb.db.RedisClient", 
+    "redis"        : "com.yahoo.ycsb.db.RedisClient",
     "voldemort"    : "com.yahoo.ycsb.db.VoldemortClient",
     "couchbase"    : "com.yahoo.ycsb.db.CouchbaseClient"
 }
 
 OPTIONS = {
-    "-P file"      : "Specify workload file",
-    "-p key=value" : "Override workload property",
-    "-s"           : "Print status to stderr",
-    "-target n"    : "Target ops/sec (default: unthrottled)",
-    "-threads n"   : "Number of client threads (default: 1)",
+    "-P file"         : "Specify workload file",
+    "-p key=value"    : "Override workload property",
+    "-s"              : "Print status to stderr",
+    "-target n"       : "Target ops/sec (default: unthrottled)",
+    "-threads n"      : "Number of client threads (default: 1)",
+    "-cp path"        : "Additional Java classpath entries",
+    "--jvm-args args" : "Additional arguments to the JVM",
```

Comment:
I used `-cp` since it gets passed directly to `java`.  Seems like single dash long form is more consistent - I'll change `--jvm-args` to `-jvm-args`.